aesni/aesni-enc-ecb.h

```diff
@@ -30,7 +31,7 @@ void AES_ECB_encrypt(const unsigned char *in,  //pointer to the PLAINTEXT
         }
 }
 
-#if 0
+#if 1
```

Comment:
This also should be removed - alongside the else

Reply:
did so